### DISCUSSION TITLE
All cert updates/renewals and creations now failing

### DISCUSSION THREAD

**Discussion by jasenf** [rcl-ssl/docs#46] [General]
Hi --

Going in to do my quarterly updates to refresh the certs and I am getting errors on all counts. Trying to update and cert gives me this:

Error:Could not manually update certificate : *.xxx.xx with challenge : DNS for target Azure Key Vault + DNS. Could not fetch nonce : Object reference not set to an instance of an object.. If this error is persistent, please delete this certificate and re-create a new one.

Tried fully deleting one and creating a new one (Standalone SAN and AzureKV+DNS).. both get this: "ERROR : Could not place order, Could not fetch nonce : Object reference not set to an instance of an object." error message

Our certs expire tomorrow (I know, last minute, but we have been doing this quarterly for years with no issues).. please help.

**Comment by jasenf** [comment]
Tried again this morning, getting oddly different messages when creating a **standalone** (not what I want, but had to try): ERROR: Could not convert Acme Order : https://acme-v02.api.letsencrypt.org/acme/order/83426023/286005215997. Could not generate validation tokens for order https://acme-v02.api.letsencrypt.org/acme/order/83426023/286005215997. Could not fetch nonce : Object reference not set to an instance of an object.

and the Azure DNS+KV one gave me this: Could not create Azure Key Vault + DNS SAN certificate : xxxx.com,*.xxxx.com . Could not convert Acme Order : https://acme-v02.api.letsencrypt.org/acme/order/83426023/286004839217. Could not generate validation tokens for order https://acme-v02.api.letsencrypt.org/acme/order/83426023/286004839217. Could not fetch nonce : Object reference not set to an instance of an object.

So now it seems its not about an order, but about validation tokens.

**Comment by rcladmin** [comment]
I checked this issue. I did not get an error on my end. It is strange that the error is changing on different attempts. It may be a temporary failure or maintenance of the Let's Encrypt Acme Server. You can try this to replace your expired certificate as a emergency fix.

- Create a Stand Alone SAN certificate to use for this emergency
- Use that  certificate the meantime to maintain SSL on your website
- Wait a day or two and delete the Stand Alone SAN certificate used for emergency
- Try creating your Azure DNS + Keyvault SAN Certificate

Please be careful when creating certificates, if there a too many failures in certificate creation you can hit the Let's Encrypt limits and certificate creation may be disabled for a week.

If errors persists, please respond ASAP, so I can assist.

**Reply by jasenf** [reply]
Hi.  Thanks for the quick response.   We are unable to create a standalone as well.  See the error message above where I say it is related to the standalone.

Thanks

**Comment by rcladmin** [comment]
I saw you were able to create : netxxxxxx.co.uk,*.netxxxx.co.uk and airxxx-api.co.uk,*.airxxx-api.co.uk . You can use those certificates in the meantime so you do not loose SSL on your website. Later on, you can delete these certificates and try creating your Azure DNS + KeyVault SAN certificate.

**Reply by jasenf** [reply]
I don't know what those domains are they have no relation to my account, sorry.

**Reply by rcladmin** [reply]
the 'xxx' is used to block out your domains in this public forum. If you are still unclear, I can list the two certificates that you were able to create today if that is ok by you. You can install those certificates on your websites in the meantime to maintain SSL on your websites.
